paddle/phi/kernels/multi_tensor_adam_utility_kernel.cuh

```diff
@@ -0,0 +1,139 @@
+#include "paddle/phi/backends/gpu/gpu_context.h"
+#include "paddle/phi/core/kernel_registry.h"
+#include "paddle/fluid/framework/tensor_util.h"
+#include "paddle/phi/core/tensor_utils.h"
+
+#include <assert.h>
+
+namespace phi {
+
+#define MAX_COMPUTE_GROUP_SIZE 65535
```

Comment:
Do not use macro. Use `constexpr` inside the function `multi_tensor_adam_utility`.